lib/configParser.ts

```diff
@@ -100,19 +103,41 @@ export class ConfigParser {
     }
 
     Object.keys(config.suites || {}).forEach((suite) => {
-      union(specs, makeArray(config.suites[suite]));
+      union(specs, getSuiteSpecs(config.suites[suite]));
     });
     return specs;
   }
 
   /**
+   * Run alterCapabilities() if it is defined for the current suite.
+   *
+   * @param {Config} config The config file
+   * @return {Capabilities} The capabilities for the suite
+   */
+  static getSuiteCapabilities(config: Config): Array<Capabilities> {
+    let suite: any = config.suite && config.suites && config.suites[config.suite];
```

Comment:
This doesn't account for input like `--suite=suite1,suite2`. I'm not even sure what the correct behavior would be in that situation.